src/reanimated2/Hooks.js

```diff
@@ -270,16 +271,19 @@ function styleUpdater(viewDescriptor, updater, state, maybeViewRef) {
   state.last = Object.assign({}, oldValues, newValues);
 
   if (Object.keys(diff).length !== 0) {
-    updateProps(viewDescriptor, diff, maybeViewRef);
+    updateProps(viewDescriptor, diff, maybeViewRef, adapters);
   }
 }
 
-export function useAnimatedStyle(updater, dependencies) {
+export function useAnimatedStyle(updater, dependencies, adapters) {
   const viewDescriptor = useSharedValue({ tag: -1, name: null }, false);
   const initRef = useRef(null);
   const inputs = Object.values(updater._closure);
   const viewRef = useRef(null);
+  // todo: should it be possible to attach multiple adapters?
+  adapters = !adapters || Array.isArray(adapters) ? adapters : [adapters];
 
+  // todo: add adapters to dependencies?
```

Comment:
We need to add adapters to the deps in order to take them into account during a fast refresh.

Reply:
yes, did it here 773657a